admin_manual/installation/server_tuning.rst

```diff
@@ -136,51 +156,49 @@ Revalidation
+Nextcloud, as much as possible, handles cache revalidation internally when required. However, this is not foolproof. In a default PHP environment, revalidation is enabled, and cached scripts are checked for changes on disk every ``2`` seconds. In many environments, these default values are reasonable and may never need to be changed.
 
 However, the revalidation frequency can be adjusted and may *potentially* enhance performance. We make no recommendations here about appropriate values for revalidation (other than the PHP defaults).
 
 .. danger::
-    Lengthening the time between revalidation (or disabling it completely) means that manual changes to scripts, including ``config.php``, will take longer before they become active (or will never do so, if
-    revalidation is disabled completely). Lengthening also increases the likelihood of transient server and application upgrade problems. It also prevents the proper toggling of maintenance mode.
-    
+    Increasing the time between revalidations (or disabling it completely) means that changes to scripts, including ``config.php``, will take longer to become active (or may never do so if revalidation is disabled completely). Increasing the interval also raises the risk of transient server and application upgrade problems and prevents the proper toggling of maintenance mode.
+
 .. warning::
-    If you adjust these parameters, you are more likely to need to restart/reload your web server (mod_php) or fpm after making configuration changes or performing upgrades. If you forget to do so, you 
-    will likely experience unusual behavior due to a mismatch between what is on disk and is in memory. These may appear to be bugs, but will go away as soon as you restart/reload mod_php/fpm.
+    If you adjust these parameters, you are more likely to need to restart/reload your web server (``mod_php``) or PHP-FPM after making configuration changes or performing upgrades. If you forget to do so, you may experience unusual behavior due to a mismatch between what is on disk and what is in memory. These may appear to be bugs, but will go away as soon as you restart/reload ``mod_php`` / fpm.
 
-To change the default from ``2`` and check for changes on disk at most every ``60`` seconds, use the following setting:
+To change the default from ``2`` and check for changes on disk at most every ``60`` seconds, add the following setting to your ``php.ini`` file:
 
 .. code:: ini
 
   opcache.revalidate_freq = 60
 
-To disable the revalidation completely:
+Alternatively, you can disable the revalidation completely:
 
 .. code:: ini
 
   opcache.validate_timestamps = 0
 
-Any Server/app upgrades or changes to ``config.php`` will then require restarting PHP (or otherwise manually clearing the cache or invalidating this particular script).
+Any server or app upgrades, or changes to ``config.php``, will then require restarting PHP (or otherwise manually clearing the cache or invalidating this particular script).
```

Comment:
Nextcloud and app upgrades trigger a full OPcache invalidation in the meantime. So this should apply only for cases where `config.php` is edited manually, config changes, app- or NC upgrades are done via CLI instead of web UI.

But this is probably something to address in a dedicated PR, which somehow makes the point clearer that the web UI and CLI use different OPcache instances, without encouraging readers too lightly to touch these settings. But anything done via web UI should never require a manual PHP restart/OPcache invalidation. Only changes done via CLI or to PHP files directly will do so, depending on (re)validation settings.